src/Akeneo/Platform/Job/back/Infrastructure/Query/SearchJobExecution.php

```diff
@@ -129,6 +131,40 @@ private function buildQueryParamsTypes(): array
         ];
     }
 
+    private function buildSqlOrderByPart(SearchJobExecutionQuery $query): string
+    {
+        $sortDirection = $query->sortDirection;
+
+        if (!in_array($sortDirection, ['ASC', 'DESC'])) {
+            throw new InvalidArgumentException(sprintf('Sort direction "%s" is not supported', $query->sortDirection));
```

Comment:
```suggestion
            throw new \InvalidArgumentException(sprintf('Sort direction "%s" is not supported', $query->sortDirection));
```
And remove it from phpcd